email-templates/src/templates/grant-approved-ticket-travel-accommodation.html

```diff
@@ -0,0 +1,187 @@
+                        we will allocate your grant to another person.
+                      </p>
+
+                      <p class="m-0 mb-4 font-semibold">Free ticket</p>
+                      <p class="m-0 mb-16">
+                        Personalized coupons will be generated to claim a free
+                        ticket.
+                      </p>
+
+                      <p class="m-0 mb-4 font-semibold">Travel grant</p>
+                      <p class="m-0 mb-16">
+                        All grants involving reimbursements will be reimbursed
+                        based on approved receipts (plane/bus/train ticket) sent
+                        to us before the conference. The reimbursement will be
+                        made by a bank transfer after the conference.
+                      </p>
+                      <p class="m-0 mb-4 font-semibold">Please note:</p>
+                      <ul class="m-0 mb-16">
+                        <li>Not approved invoices will not be reimbursed.</li>
+                        <li>
+                          Reimbursements before the conference are not possible.
+                        </li>
+                        <li>
+                          The financial aid grants will only be made available
+                          to registered attendees actually attending the
+                          conference.
+                        </li>
+                        <li>Transfers to other people are not possible.</li>
+                        <li>
+                          As part of our efforts to reduce the carbon footprint
+                          of the conference we expect our attendees to use
+                          trains rather than airplanes when possible. Thus, we
+                          will not refund air travel costs when other means of
+                          transportation are easily available. This will be
+                          considered on a case by case basis. Please consider
+                          this when calculating the estimated costs.
+                        </li>
+                      </ul>
+                      <p class="m-0 mb-16">
+                        If you need an invitation letter for a VISA requirement
+                        please, click on the link above and write with your
+                        personal details (complete name/surname, birthdate,
+                        nationality, passport number) and we'll send you the
+                        invitation letter within 4 days.
+                      </p>
+                      <p class="m-0 mb-4 font-semibold">Volunteering</p>
+
+                      <p class="m-0 mb-16">
+                        In the application form you stated you'd like to help
+                        the organization. First of all, thank you for your
+                        interest in our conference 😊
+                      </p>
```

Comment:
should this inside a if? what if I didn't say I want to volunteer? or do we assume they all did?